themes/dohmh/layouts/data-explorer/single.html

```diff
@@ -2773,20 +2773,26 @@ <h6>${secondaryIndicatorName} - ${secondaryMeasure}</h6>
         btn.innerHTML = `<i class="fas fa-copy mr-1"></i>Copied!`
         // console.log('citation copied!')
     }
-    
+
     createCitation();
-    
+
     // export current table view
 
     $("#thisView").on("click", (e) => {
```

Comment:
@mmontesanonyc this is the download we are not able to pass along the file name to GA

Reply:
Might be possible to populate this file name with the name of the selected indicator - I wonder if it's OK to leave this as a placeholder for now.